open_city_profile/settings.py

```diff
@@ -35,7 +35,7 @@
     TOKEN_AUTH_ACCEPTED_AUDIENCE=(str, ""),
     TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX=(str, ""),
     TOKEN_AUTH_REQUIRE_SCOPE=(bool, False),
-    TOKEN_AUTH_AUTHSERVER_URL=(str, ""),
+    TOKEN_AUTH_AUTHSERVER_URL=(list, []),
```

Comment:
Having a simple list of authorization servers probably isn't good enough. For example #247 starts using the authorization server(s) for other purposes than just passing the configuration directly to django-helusers.

Perhaps we need one main authorization server for use cases where the authorization server is called directly and zero or more additional authorization servers that are accepted as access token providers.

Reply:
Ok by me. Should be simple to implement at this point? If you mean just adding one more ENV variable + setting. I guess it would be nice to have a programmatic "helper" ENV variable which cointains the contents of both?